build/xtask/src/config.rs

```diff
@@ -348,6 +348,58 @@ impl Config {
             );
         }
 
+        if let Some(ext_int_ctrl_toml) = self.tasks.get("ext_int_ctrl") {
```

Comment:
How is it decided whether some functionality ends up in build/xtask directly or added as a function `expose_something_something` in build_util? I am wondering if this interrupt notifications thing is something that should be part of build_util. Can it be simply generated by those tasks that require this information?

Reply:
Most of the `build_util` functions deserialize some environment variable created in `xtask`. Instead of creating a `String`, `xtask` could create and serialize a struct with all this information that `build_util::task_irq_consts()` uses to generate the same constants. That would probably save a bit on runtime costs for the build, but otherwise would just add complexity.

Reply:
Ah okay. I was mostly looking at it from the perspective of keeping RV specific bits out of common xtask code. But, if this seems like the right place considering the complexity, then I think its good.